page/new-innovations.php

```diff
@@ -0,0 +1,109 @@
+<link rel="stylesheet" href="css/template-1.css">
+<div class="full text-center">
+    <span class="garis"></span>
+    <h1 class="display-5 text-light"><dt><b>INFORMASI PENGADAAN</b></dt></h1>
+</div>
+
+<div class="container mt-5">
+    <div class="row">
+        <div class="col-md-8">
+
+            <?php
+                $select = mysqli_query($connection, "select * from berita where kategori='new-innovation'");
+                while($data = mysqli_fetch_array($select)){
+                    if($data[4]==null || $data[4]==""){
+                        $thumbnail = "#";
+                    }else{
+                        $thumbnail = "news/".explode("-", $data[4])[0]."/gambar/".explode("-", $data[4])[1];
+                    }
+                    ?>
+
+                    <div class="row article-berita mb-5 pb-5">
+                        <div class="col-md-2 text-center">
+                            <div class="tanggal-group float-right">
+                                <div class="hari"><?php echo date("d", strtotime($data[2])) ?></div>
```

Comment:
Codacy found an issue: [All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'date'.](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5762293)